### DISCUSSION TITLE
Notifications only for subscribed forums and forum discussions

### DISCUSSION THREAD

**Discussion by sourfe** [muraliwebworld/push-notification-plugin-for-wordpress-websites#21] [Ideas]
If I enable notifications for forum posts I get notifications from all forums and forums posts which can be quite a lot for a busy site.
Much better if that can be filtered down to only subscribed.

**Comment by muraliwebworld** [comment]
I remembe i added that option for onesignal with buddyboss last year, I will check, I will update it in next release early next week

**Reply by sourfe** [reply]
Do you have anymore info about this working also for Firebase?

**Reply by muraliwebworld** [reply]
I will add this for firebase also for both buddypress and buddyboss. I need to check bbpress plugin which field it is storing, accordingly i will add this option, i am preparing for this week release, i will include this change also

**Comment by sourfe** [comment]
Thanks for the update
